lightning_pose/models/base.py

```diff
@@ -177,7 +177,6 @@ def __init__(
 
         """
         super().__init__()
-        print(f"\n Initializing a {self._get_name()} instance.")
```

Comment:
let's keep this, I find it a helpful output from the command line. does it interact poorly with the multi-gpu setup?

Reply:
No issues, I just thought it was a leftover to be cleaned up. I'll put it back.

Aside - Multi-GPU print statements will appear multiple times unless you guard the code by an "is rank zero" check. i'll add it while I'm here.